crm_project_task/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,4 @@
+To configure this module, you need to:
+
```

Comment:
- Create a Project with right configuration in order that all salesman have access (usualy all employees)

@EmilioPascual see

![imagen](https://github.com/OCA/crm/assets/8736623/54498103-7c9f-48e5-a9a4-ed2278d558f5)

Reply:
Done